src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -79,19 +79,15 @@
 </table>
 
 <div class="ez-list-toolbar">
-    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.subitems.default.ordering'|trans()|desc('Sub-items default ordering') }}</h2>
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.sub.items.sorting.order'|trans()|desc('Sub-items sorting order') }}</h2>
 </div>
 
-<p class="ez-subitem-ordering-details">{{ 'locationview.details.default.listing.of.subitems.by'|trans()|desc('Default listing of sub-items by') }}
-    <select class="ez-subitems-ordering-sort-type" disabled>
-        {% for fieldId, fieldLabel in ordering.sortFields %}
-        <option {% if location.sortField == fieldId %}selected{% endif %} value="{{ fieldId }}">{{ fieldLabel.key|trans()|desc(fieldLabel.default) }}</option>
-        {% endfor %}
-    </select>
+{{ form_start(orderingForm, {'action': path('ez_hybrid_platform_ui_location_update_sub_items_order', {'locationId': location.id, 'contentId': contentInfo.id})}) }}
```

Comment:
to be consistent with others forms, the `<form>` tag should start right before the `<div class="ez-list-toolbar">` element